### PR TITLE
Cleaning

### PR DESCRIPTION
**Pull request by jgiovaresco**

### CODE REVIEW

File: src/Etats/edit.ts [hunk 1]
```diff
@@ -439,7 +435,7 @@ export const Edit = {
         return [rayon, degrees];
     },
     redimensionne: function(s) {
-        // Redimensionnement de la fenêtre en fonction de la hauteur (rayon)
+        // Redimensionnement de la fenêtre en fonction de la PLATFORMS_HIGHT (rayon)
```
Comment (Remi-p) [inline src/Etats/edit.ts]: Maybe leave the comment as is, for now?

File: src/Etats/play.ts [hunk 2]
```diff
@@ -23,8 +23,7 @@ let s_invers = 1 / s;
 let fps_adapt = 1;
 const fps_array = [];
 
-// Animation de fin et de début
-let anim = true;
+let levelStarted = false;
```
Comment (Remi-p) [inline src/Etats/play.ts]: Maybe `isLevelPlaying` as it also seems to be `false` on the end of the level 🤷‍♂ ?

File: src/Etats/play.ts [hunk 3]
```diff
@@ -486,7 +485,7 @@ export const Play = {
 
             // ------------- Animation de fin de niveau //
             this.dude.body.gravity.y = 0;
-            anim = true;
+            levelStarted = false;
```
Comment (Remi-p) [inline src/Etats/play.ts]: No endgame animation 😨?

File: src/Etats/edit.ts (hunk 1 above)
Comment (jgiovaresco) [reply src/Etats/edit.ts]: 🤔 wrong replace :p

File: src/Etats/play.ts (hunk 2 above)
Comment (jgiovaresco) [reply src/Etats/play.ts]: Agree it is a better name ;)

**Pull request merged**